client/extensions/woocommerce/app/products/product-form-simple-card.js

```diff
@@ -87,14 +95,17 @@ const ProductFormSimpleCard = ( { siteId, product, editProduct, translate } ) =>
 		'products__product-form-stock-disabled': ! product.manage_stock,
 	} );
 
+	const { stock_quantity } = product;
+	const quantity = ! isNull( stock_quantity ) && Number( stock_quantity ) >= 0 ? stock_quantity : '';
```

Comment:
Good call. Yeah stock can go negative for backorders. The whole line in general was to make sure we could display 0 and still have the other stock options display, and hide if no value is entered (per design) but the `isNull` check already takes care of that. I've removed the negative stock check. Thanks.